test/parallel/test-timers-unenroll-unref-interval.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const common = require('../common');
+const timers = require('timers');
+
+const keepOpen = setTimeout(() => {
+  common.fail('Test timed out.');
+}, common.platformTimeout(500)); // Keep event loop open.
```

Comment:
I guess I'll also add/clarify that even If it seems like I'm being ridiculous and this will never ever cause problems itself, arbitrary-length timeouts in tests are a bit of an anti-pattern that have bitten us over and over. So even if it's just to set a good example for other people writing tests, it's good (IMO) to remove arbitrary-length timers.